spec/classes/puppet_server_puppetserver_spec.rb

```diff
@@ -100,6 +101,10 @@
             content = catalogue.resource('file', '/etc/custom/puppetserver/conf.d/puppetserver.conf').send(:parameters)[:content]
             expect(content).to include(%Q[    master-var-dir: /opt/puppetlabs/server/data/puppetserver\n])
           end
+          it 'should have max-requests-per-instance: /opt/puppetlabs/server/data/puppetserver' do
```

Comment:
I know it's a bit overengineered, but please add a seperate test for this with a non-zero value and check for the zero here.